build-packages.sh

```
@@ -1,3 +1,5 @@
+set -euo pipefail
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)